src/styles/blocks/todoapp.scss

```diff
@@ -0,0 +1,166 @@
+.todoapp {
+  margin: $appSideMargin 20px;
+  color: #4d4d4d;
+
+  &__title {
+    margin: 0;
+
+    text-align: center;
+    font-size: 100px;
+    font-weight: 100;
+    color: rgba(175, 47, 47, 0.15);
+    -webkit-text-rendering: optimizeLegibility;
+    -moz-text-rendering: optimizeLegibility;
+    text-rendering: optimizeLegibility;
+  }
+
+  &__userTitle {
+    text-align: center;
+    color: rgba(175,47,47,0.25);
+    font-size: 24px;
+    font-style: italic;
+  }
+
+  &__content {
+    margin-bottom: 20px;
+
+    background: #fff;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2),
+      0 25px 50px 0 rgba(0, 0, 0, 0.1);
+  }
+
+  &__header {
+    position: relative;
+  }
+
+  &__toggle-all {
+    position: absolute;
+    right: 100%;
+    bottom: 100%;
+
+    width: 1px;
+    height: 1px;
+    border: none; /* Mobile Safari */
+    opacity: 0;
+
+    @media screen and (-webkit-min-device-pixel-ratio:0) {
+      background: none;
+    }
+  }
+
+  &__toggle-all-label {
+    position: absolute;
+    top: -52px;
+    left: -13px;
+
+    width: 60px;
+    height: 34px;
+    font-size: 0;
+    -webkit-transform: rotate(90deg);
+    transform: rotate(90deg);
+
+    &:before {
+      padding: 10px 27px;
+
+      content: "❯";
+      font-size: 22px;
+      color: #e6e6e6;
+    }
+  }
+
+  &__toggle-all:checked + &__toggle-all-label:before {
+    color: #737373;
+  }
+
+  &__new-todo {
+    position: relative;
+
+    margin: 0;
+    padding: 16px 16px 16px 60px;
+    width: 100%;
+    border: none;
+    box-sizing: border-box;
+
+    font-size: 24px;
+    font-family: inherit;
+    font-weight: inherit;
+    line-height: 1.4em;
+    color: inherit;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+
+    &:focus {
+      outline: none;
+    }
+
+    &::placeholder {
+      font-style: italic;
+      font-weight: 300;
+      color: #e6e6e6;
+    }
+  }
+
+  &__main {
+    position: relative;
+    z-index: 2;
+
+    border-top: 1px solid #e6e6e6;
```

Comment:
Consider using a variable for the color #e6e6e6 to make it reusable and easy to maintain.